docs/faq/adoption.md

```diff
@@ -0,0 +1,7 @@
+---
+id: Adoption FAQs
+title: Adoption FAQs
+description: All FAQs related to Adoption
```

Comment:
Let's skip the ones we don't have any content for yet and leave them as future additions